packages/cron/src/jobs/storage.js

```diff
@@ -44,40 +71,51 @@ const STORAGE_QUOTA_EMAILS = [
  * Get users with storage quota usage in percentage range and email them as
  * appropriate when approaching their storage quota limit.
  * @param {{
- *  db: import('@web3-storage/db').DBClient
+ *  roPg: import('pg').Client
  *  emailService: import('../lib/email/service').EmailService
+ *  userBatchSize?: number
  * }} config
  */
-export async function checkStorageUsed ({ db, emailService }) {
+export async function checkStorageUsed ({ roPg, emailService, userBatchSize = 100 }) {
```

Comment:
This could possibly be bigger, I wonder if 500 or a 1000 would be better?

Reply:
A note here: we rely on user id's that came from Fauna. The bad news is that they are not sequential, so there are a lot of gaps, which is really annoying for this kind of use cases...

With that in mind, there are huge gaps sometimes between ids. So, I think we should start with 1000 as you suggest

Reply:
Updated! 👍